docs/tools/plateformio.mdx

```diff
@@ -0,0 +1,90 @@
+---
+custom_edit_url: null
+description: The gate is a major tool of the Luos ecosystem.
+---
+
+import { customFields } from '/docusaurus.config.js';
+import Tooltip from '@site/src/components/Tooltip.js';
+import Image from '@site/src/components/Images.js';
+
+# Plateformio
+
+## What is plateformio?
+
+Plateformio is user-friendly and extensible integrated development environment with a set of professional development instruments, compiler and debugger. Plateformio provide modern and powerful features to simplify the creation and delivery of embedded products
+That's the reason why Luos use plateformios project structure for its examples, allowing user to use and share easily service and development on a unified environment. Also luos_engine and some tools like [Gate](/docs/tools/gate) or [Inspector](/docs/tools/inspector) are available on plateformios registry! This registry allow user to link directly a project and a library, like luos_engine, by including dependencies in the configuration project file call plateformio.ini
+learn more about <a href="https://platformio.org" target="_blank">Plateformio and how to install Plateformio<IconExternalLink width="10"></IconExternalLink></a>.
+
+## Plateformio project structure
+
+Every plateformio project can have this folder project structure and must have at the root of the project a platformio.ini file
+
+```AsciiDoc
+ Project
+    │
+    ├─── lib
+    ├─── include
+    ├─── src
+    └─── platformio.ini
+```
+|      Folder      |                 Description             |
+| :--------------: |---------------------------------------: |
+|       lib        |    The local library for the project    |
+|     include      |      Header files of your project       |
+|        src       |      Source files of your project       |
+|  platformio.ini  |  Plateformio project configuration file |
```

Comment:
This is pure pio things, we don't want to teach PIO here. You should redirect to PIO documentation instead.